<launch>
  <include file="$(find pr2_machine)/$(env ROBOT).machine" />
  <rosparam command="load" file="$(find pr2_moveit_config)/config/kinematics.yaml"/>

  <!-- Move it -->
  <include file="$(find pr2_moveit_config)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>
   <include file="$(find pr2_moveit_config)/launch/move_group.launch">
    <arg name="allow_trajectory_execution" value="true"/>
  </include>
  <include file="$(find pr2_moveit_config)/launch/default_warehouse_db.launch" />
  <!-- End move it -->

  <param name="leash_length" value="1.25"/>

  <node pkg="dogsim" type="get_path_server" name="get_path_server" output="screen"/>

  <node name="map_server" pkg="map_server" type="map_server" args="$(find gazebo_worlds)/Media/materials/textures/map_blank.png 0.1" respawn="true" />
 
  <node pkg="dogsim" type="map_broadcaster" name="map_broadcaster" output="screen"/>

  <node pkg="dogsim" type="robot_driver" name="robot_driver" output="screen">
    <param name="solo_mode" value="true"/>
  </node>

  <node pkg="dogsim" type="move_robot_action" name="move_robot_action" output="screen"/>

  <node pkg="dogsim" type="move_dog_away_action" name="move_dog_away_action" output="screen"/>

  <node pkg="pr2_tuckarm" type="tuck_arms.py" name="tuck_arms_app" output="screen" args="-l tuck -r untuck"/>

 <node pkg="dogsim" type="move_arm_to_base_position_action" name="move_arm_to_base_position_action" output="screen"/>

  <node pkg="dogsim" type="path_visualizer" name="path_visualizer" />

</launch>
